src/databricks/labs/ucx/install.py

```diff
@@ -893,13 +895,28 @@ def repair_run(self, workflow):
                 return
             latest_job_run = job_runs[0]
             state = latest_job_run.state
+
+            while not state.result_state and (time.time() - start_time < timeout):
```

Comment:
Refactor this into private method and decode with retried

Reply:
@nfx .Refactored the same  with retried logic.